vega_sim/scenario/comprehensive_market/agents.py

```diff
@@ -0,0 +1,33 @@
+from collections import namedtuple
+
+WalletConfig = namedtuple("WalletConfig", ["name", "passphrase"])
+
+# Set up wallets for parties submitting / amending liquidity provisions
+MM_WALLET = WalletConfig("mm", "pin")
+
+BG_WALLET = WalletConfig("bg", "pin")
+
+# Set up wallets for parties submitting market orders
+MO_WALLETS = (
+    WalletConfig("mo_trader_a", "pass"),
+    WalletConfig("mo_trader_b", "pass"),
+    WalletConfig("mo_trader_c", "pass"),
+    WalletConfig("mo_trader_d", "pass"),
+    WalletConfig("mo_trader_e", "pass"),
+)
```

Comment:
It would be good if we made the number of these a customisable parameter on the scenario. We can just use a base username then shove a number on the end to generate.

Reply:
Sorry yes was definitely the plan, haven't finished all the changes for this scenario yet.